sfxclient/sfxclient.go

```diff
@@ -372,3 +378,41 @@ func (s *Scheduler) Schedule(ctx context.Context) error {
 		}
 	}
 }
+
+func (s *Scheduler) checkDatapoints(ret []*datapoint.Datapoint) {
+	if len(os.Getenv(panicOnDupes)) > 0 {
+		var dupes []*datapoint.Datapoint
+		m := make(map[uint32]*datapoint.Datapoint, len(ret))
+		buff := bytes.Buffer{}
+		for _, d := range ret {
+			buff.Reset()
+
+			buff.WriteString(d.Metric)
```

Comment:
Given that we sometimes use 1s resolution you might as well include timestamp in this just so it never is an issue in case there is batching of input datapoints to the client that could cause multiple dps in the same MTS but different timestamps to be in the same batch (this was totally possible with the agent if there was a slowdown somewhere).